deployment/README.md

```diff
@@ -14,7 +14,7 @@ $ sudo ./build.sh
 
 **NOTE:**
   - CCNP kernel patches are at [kernel](../tools/build/kernel/)
-  - The tool should be run on a Ubuntu 23.10 TDX host with TDX early preview packages installed. Please refer to [here](https://github.com/canonical/tdx)
+  - The tool should be run on a Ubuntu 24.04 TDX host with TDX early preview packages installed. Please refer to [here](https://github.com/canonical/tdx)
```

Comment:
May we also add a note here to notice users that using ubuntu 23.10 to use the previous release?

Reply:
There is a version mapping in the table [here](https://github.com/cc-api/confidential-cloud-native-primitives?tab=readme-ov-file#configuration).
But good suggestion, let me also add it here.

Reply:
Updated. Thanks.